src/common/prop.c

```diff
@@ -73,30 +96,77 @@ daos_prop_free(daos_prop_t *prop)
 		struct daos_prop_entry *entry;
 
 		entry = &prop->dpp_entries[i];
-		switch (entry->dpe_type) {
-		case DAOS_PROP_PO_LABEL:
-		case DAOS_PROP_CO_LABEL:
-		case DAOS_PROP_PO_OWNER:
-		case DAOS_PROP_CO_OWNER:
-		case DAOS_PROP_PO_OWNER_GROUP:
-		case DAOS_PROP_CO_OWNER_GROUP:
-			if (entry->dpe_str)
-				D_FREE(entry->dpe_str);
-			break;
-		case DAOS_PROP_PO_ACL:
-		case DAOS_PROP_CO_ACL:
-			if (entry->dpe_val_ptr)
-				D_FREE(entry->dpe_val_ptr);
-			break;
-		default:
-			break;
-		};
+		daos_prop_entry_free_value(entry);
 	}
 
 	D_FREE(prop->dpp_entries);
 	D_FREE_PTR(prop);
 }
 
+daos_prop_t *
+daos_prop_merge(daos_prop_t* old_prop, daos_prop_t *new_prop)
```

Comment:
(style)  "foo* bar" should be "foo *bar"

Reply:
daosbuild1 is right this time. :) No need to refresh just for this though.

Reply:
I did fix this, just didn't note it when I did it.